packager/debian/postinst

```diff
@@ -30,4 +30,6 @@ case "$1" in
     ${CLI} scale web=1 || true
   ;;
 
+service gogs restart || true
+
```

Comment:
I'm no bash expert, but why did you write this line out of all the other cases? I mean, shouldn't it go before the `;;` at line 31?